client/classic/upe/index.js

```diff
@@ -163,18 +93,6 @@ jQuery( function ( $ ) {
 		$form.removeClass( 'processing' ).unblock();
 	};
 
-	/**
-	 * Checks whether SEPA IBAN element is present in the DOM and needs to be mounted
-	 *
-	 * @return {boolean} Whether IBAN needs to be mounted
-	 */
-	const doesIbanNeedToBeMounted = () => {
-		return (
-			$( '#stripe-iban-element' ).length &&
-			! $( '#stripe-iban-element' ).children().length
-		);
-	};
-
```

Comment:
@cesarcosta99 Stripe made changes to how UPE works since the beta.

In order to [upgrade](https://stripe.com/docs/payments/payment-element/upgrade-from-beta) from the beta flag, I had reverted #1767. Everything seems to be working as expected for SEPA, but It would be nice if you could double-check/review.

Reply:
I tested all items in the UPE section you wrote in the testing instructions with both credit card and SEPA. It worked well 👍